Gemfile.lock

```diff
@@ -369,4 +366,4 @@ DEPENDENCIES
   yabeda-rails
 
 BUNDLED WITH
-   2.0.2
```

Comment:
This is ok, but JFYI the bundler version of the productization process is 2.0.2. So it will complain. 

In the toolbox we also `Bundled With` 2.1.4 with no issues